renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -203,6 +203,13 @@ class EGLHook : LibraryHook, public GLPlatform
     GLWindowingData ret;
     EGLNativeWindowType window = 0;
 
+#if ENABLED(RDOC_ANDROID)
+    if(system == eWindowingSystem_Android)
+    {
+        window = (EGLNativeWindowType)data;
+    }
+    else
```

Comment:
If it's possible I'd like to have support on android for window-less contexts if system is `eWindowingSystem_Unknown`. On desktop this just creates a context on a pbuffer without any window attached, can you do that a similar way on EGL?

Doesn't have to be fixed this PR, but would be nice as a follow-up.